traitsui/testing/tester/_ui_tester_registry/qt4/_traitsui/button_editor.py

```diff
@@ -31,7 +31,8 @@ def register(registry):
             lambda wrapper, _: _interaction_helpers.mouse_click_qwidget(
                 wrapper._target.control, wrapper.delay)),
         (DisplayedText,
-            lambda wrapper, _: wrapper._target.control.text())
+            lambda wrapper, _: wrapper._target.control.text()),
+        (IsEnabled, lambda wrapper, _: wrapper._target.control.isEnabled())
```

Comment:
```suggestion
        (IsEnabled, lambda wrapper, _: wrapper._target.control.isEnabled()),
```
nit-picky comment : leaving a trailing comma on the last item in a list ensures that we won't see a diff like this one the next time.

Reply:
comment applies here and elsewhere in the PR.